lib/ccavenue-sdk.rb

```diff
@@ -363,11 +363,12 @@ def order_status_updated_at
     end
 
     ### merchant credentials validation
-    VALIDATION_SUCCESS_MSG = 'Providing Reference_No/Order No is mandatory'
+    VALIDATION_SUCCESS_MSGS = ['Providing Reference_No/Order No is mandatory',
+                              'Providing Reference number/Order Number is mandatory']
 
     def credentials_valid?
-      valid_reason = @reason.match(/#{VALIDATION_SUCCESS_MSG}/) unless @reason.blank?
-      self.http_status == :success && self.api_status == :success && valid_reason.present?
+      valid_reason = VALIDATION_SUCCESS_MSGS.include?(@reason)
```

Comment:
Because this is last in the validation on line 371, we should move this down and save the evaluation if not needed.  I will make the change.